### PR TITLE
Make pull request title changeable

### PR DESCRIPTION
**Pull request by sachin21**
# Summary

I want to change title when tachikoma was sent a pull request.

## Tasks

- [x] Make pull request title changeable.
- [x] Write a document for the key of `pull_request_title.

### Before

selection_041

### After

selection_042

## Ask
- [x] Should I write a changelog? If so, tell me next version's number.

### CODE REVIEW

File: data/default.yaml [hunk 1]
```diff
@@ -39,6 +39,12 @@ timestamp_format: '%Y%m%d%H%M%S'
 # https://github.com/sanemat/gist-mail/pull/54
 #
 pull_request_body: ':hamster::hamster::hamster:'
+
+# The pull request title.
+# You can change pull request title. Default is `Exec tachikoma update %s`
+# %s will be setted timestamp.
+#
+pull_request_body: "Execute `bundle update` at %s"
```
Comment (sanemat) [inline data/default.yaml]: `pull_request_title`?

Comment (sachin21) [inline data/default.yaml]: Oh, my bad.

Comment (sachin21) [inline data/default.yaml]: Thanks, fixed and force pushed.

**Comment by sachin21** [comment]
Kicked houndci-bot. I didn't change anything.

**Comment by sachin21** [comment]
@sanemat Hi, Could you merge this pull request?

**Comment by sanemat** [comment]
Sorry for the delayed response. I merged this PR :+1:

**Comment by sachin21** [comment]
@sanemat Thank you very much! Thank for your time.

**Pull request merged**